Description: Fix -Waggressive-loop-optimizations compiler warning.
 In particular, the cpu_history array has length 59, so when i = 58,
 cpu_history[i+1] is undefined.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-02-02

--- a/src/wmgtemp.c
+++ b/src/wmgtemp.c
@@ -428,7 +428,7 @@
   double sys_high = highest_temp(sys_history);
 
   /* Shift the arrays */
-  for(i = 0; i < 59; i++) {
+  for(i = 0; i < 58; i++) {
     cpu_history[i] = cpu_history[i + 1];
     sys_history[i] = sys_history[i + 1];
   }
